lib/helpers/specification.js

```diff
@@ -111,21 +113,32 @@ function organizeSwaggerProperties(swaggerObject, pathObject, propertyName) {
     'parameters',
     'definition',
     'definitions',
+    'topic',
+    'topics',
+    'baseTopic',
+    'server',
+    'servers',
+    'security',
+    'externalDocs',
   ];
 
   // Common properties.
   if (simpleProperties.indexOf(propertyName) !== -1) {
     const keyName = correctSwaggerKey(propertyName);
-    const definitionNames = Object.getOwnPropertyNames(
-      pathObject[propertyName]
-    );
-    for (let k = 0; k < definitionNames.length; k += 1) {
-      const definitionName = definitionNames[k];
-      swaggerObject[keyName][definitionName] = Object.assign(
-        {},
-        swaggerObject[keyName][definitionName],
-        pathObject[propertyName][definitionName]
+    if (propertyName === 'baseTopic') {
+      swaggerObject[propertyName] = pathObject[propertyName];
+    } else {
+      const definitionNames = Object.getOwnPropertyNames(
+        pathObject[propertyName]
       );
+      for (let k = 0; k < definitionNames.length; k += 1) {
+        const definitionName = definitionNames[k];
+        swaggerObject[keyName][definitionName] = Object.assign(
+          {},
+          swaggerObject[keyName][definitionName],
+          pathObject[propertyName][definitionName]
+        );
+      }
```

Comment:
seems a bit a too central change and an eager refactoring, due to sharing the parser